tools/not.cpp

```diff
@@ -17,14 +17,20 @@
 #include <cstdlib>
 #include <iostream>
 #include <sstream>
+#include <spawn.h>
```

Comment:
If I remember correctly `spawn.h` and is not a standard library, and not supported on Windows? you may not have access to `wait.h` on Windows as well... I saw you used `waitpid` only on IOS ifdef, so could you move it under `ifdef __APPLE__` as well?

Reply:
Done